test/jdk/java/net/httpclient/http2/ConnectionFlowControlTest.java

```diff
@@ -258,33 +240,33 @@ static void assertDetailMessage(Throwable throwable, int iterationIndex) {
         }
     }
 
-    @BeforeTest
-    public void setup() throws Exception {
+    @BeforeAll
+    public static void setup() throws Exception {
         sslContext = new SimpleSSLContext().get();
         if (sslContext == null)
             throw new AssertionError("Unexpected null sslContext");
 
-        var http2TestServer = new Http2TestServer("localhost", false, 0);
-        http2TestServer.addHandler(new Http2TestHandler(), "/http2/");
-        this.http2TestServer = HttpTestServer.of(http2TestServer);
-        http2URI = "http://" + this.http2TestServer.serverAuthority() + "/http2/x";
+        var http2TestServerLocal = new Http2TestServer("localhost", false, 0);
+        http2TestServerLocal.addHandler(new Http2TestHandler(), "/http2/");
+        http2TestServer = HttpTestServer.of(http2TestServerLocal);
+        http2URI = "http://" + http2TestServer.serverAuthority() + "/http2/x";
 
-        var https2TestServer = new Http2TestServer("localhost", true, sslContext);
-        https2TestServer.addHandler(new Http2TestHandler(), "/https2/");
-        this.https2TestServer = HttpTestServer.of(https2TestServer);
-        https2URI = "https://" + this.https2TestServer.serverAuthority() + "/https2/x";
+        var https2TestServerLocal = new Http2TestServer("localhost", true, sslContext);
+        https2TestServerLocal.addHandler(new Http2TestHandler(), "/https2/");
+        https2TestServer = HttpTestServer.of(https2TestServerLocal);
+        https2URI = "https://" + https2TestServer.serverAuthority() + "/https2/x";
 
         // Override the default exchange supplier with a custom one to enable
         // particular test scenarios
-        http2TestServer.setExchangeSupplier(FCHttp2TestExchange::new);
-        https2TestServer.setExchangeSupplier(FCHttp2TestExchange::new);
+        http2TestServerLocal.setExchangeSupplier(FCHttp2TestExchange::new);
+        https2TestServerLocal.setExchangeSupplier(FCHttp2TestExchange::new);
 
-        this.http2TestServer.start();
-        this.https2TestServer.start();
+        http2TestServerLocal.start();
+        https2TestServerLocal.start();
```

Comment:
I believe you should revert these changes or make sure they result in exactly the same calls. For instance - `http2TestServerLocal.start();` is not exactly the same as `this.http2TestServer.start();` - the original code was calling start() on the wrapper, and IMO it's better to continue doing that.